src/reducers/deposit.js

```diff
@@ -47,6 +56,12 @@ const deposit = (state = initialState, action) => {
         ...state,
         depositAddress: action.payload.depositAddress,
         invoiceStatus: 2,
+        isStateReady: true
```

Comment:
Actually I wonder if this makes sense to set at `DEPOSIT_REQUEST_BEGIN` even.

The deposit state is ready by default *unless* we just loaded a page with a deposit id in the URL. Right now I think we assume the deposit state is unready unless we specifically flag it as ready.

Still not blocking haha.